### PR TITLE
Remove unused variables

### PR DESCRIPTION
**Pull request by Crease29**

### CODE REVIEW

File: source/Application/Controller/Admin/GenericImportMain.php [hunk 1]
```diff
@@ -183,7 +183,7 @@ protected function _getCsvFieldsNames()
     {
         $blCsvContainsHeader = $this->getConfig()->getRequestParameter('blContainsHeader');
         \OxidEsales\Eshop\Core\Registry::getSession()->setVariable('blCsvContainsHeader', $blCsvContainsHeader);
-        $sCsvPath = $this->_getUploadedCsvFilePath();
+        $this->_getUploadedCsvFilePath();
```
Comment (Crease29) [inline source/Application/Controller/Admin/GenericImportMain.php]: I'm not sure about this one so I only cleared the variable assignment. The called method seems to execute something relevant which is relevant for the current method.

File: source/Application/Controller/Admin/OrderMain.php [hunk 2]
```diff
@@ -166,7 +166,7 @@ public function sendOrder()
             $oOrder->save();
 
             // #1071C
-            $oOrderArticles = $oOrder->getOrderArticles(true);
+            $oOrder->getOrderArticles(true);
```
Comment (Crease29) [inline source/Application/Controller/Admin/OrderMain.php]: I'm not sure about this one so I only cleared the variable assignment. The called method seems to execute something relevant which is relevant for the current method.

**Comment by Sieg** [comment]
Hello @Crease29, thank you for the improvements you did! I have rebased on latest b-6.x and merged it at 1459dafe753e1890924d71958bc8df353a407bf8 with some fixes from my side. Looks like some variables were still needed. Please double check if i have missed something and have a nice day! :)

**Comment by Crease29** [comment]
Looks good to me! Thank you for rebasing and merging :slightly_smiling_face:

**Pull request closed without merging**